ros/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.autosave
```

Comment:
Since this is now incorporated into the parent directory, this can go.  We should probably include relevant ignore directives into the parent's .gitignore

Reply:
Done.  Thanks.